concourse/RELEASING.md

```diff
@@ -3,7 +3,7 @@
 We publish the JVM driver on maven central. Our pipeline is driven by the
 repository itself. The runbook below describes the release process.
 
-1. When issuing a latest mainline release, create branch off of `main`.
+1. When issuing the latest mainline release, create branch off of `main`.
```

Comment:
Ah! We have a releasing readme!

Let's move this to the top of the repo, I think. Not in this PR ofc.

Reply:
Will do.